content/2017.12.kinto-at-mozilla.rst

```diff
@@ -0,0 +1,148 @@
+Kinto at Mozilla
+################
+
+:lang: en
+:date: 2017-12-20
+:url: kinto-at-mozilla
+:summary: A summary of how Kinto is used at Mozilla
+
+
+During the last company «all hands», we've heard many mentions of *Kinto*, mostly from people that we hadn't met before or that wasn't aware we were the authors. Part of this fame may probably be due to the success of Firefox Notes, and the recent addition of the long awaited synchronization features.
+
+It's amazing how we've accomplished with this project in very heterogenous contexts, and sadly we haven't really taken the time to celebrate. This post intends to present the different use-cases we are aware of at Mozilla, with some architecture and metrics details.
+
+
+Firefox remote settings
+=======================
+
+The historical one!
+
+Firefox engineers edit remote settings like blocklist or certificates revocation lists via the Kinto Admin UI. There are several buckets for each category of settings listed below.
+
+The hundreds millions of Firefox browsers regularly poll for changes using the `kinto-changes` endpoint. Two instances are deployed with the same database, a public one read-only behind a CDN, and a private one only accessible via a secured VPN.
+
+Since the remote settings are critical, and served via a CDN, we use `kinto-signer` to sign the data in order to guarantee authenticity and integrity. It also brings reviewing and sign-off features as well as previewing changes for the QA team. The history plugin allows to track the changes over time.
```

Comment:
`It also brings reviewing and sign-off features as well as previewing changes for the QA team` parses for me as `It (also brings reviewing and sign-off features) as well as (previewing changes for the QA team)`, but maybe you meant it as `It also brings (reviewing and sign-off features as well as previewing changes for the QA team)`? If so, maybe change `previewing changes` to `the previewing changes feature` to emphasize the parallelism. Also, instead of `bringing` features I would probably write `adds reviewing and sign-off features`.

`allows us to track` or `allows tracking`.